File: solutions/javascript/FirstUniqueCharacterinaString/FirstUniqueCharacterinaString.js
/**
 * @param {string} s
 * @return {number}
 */
var firstUniqChar = function(s) {
    for (let i = 0; i < s.length; i++) {
        for (let j = 0; j < s.length; j++) {
            if (i != j) {
                if (s[i] == s[j]) {
                    break;
                } else if (j == s.length - 1) {
                    return i;
                }
            } else if (j == s.length - 1) {
                return i;
            }
        }
    }
    return -1;
};

File: solutions/java/FirstUniqueCharacterinaString/FirstUniqueCharacterinaString.java
class Solution {
    public int firstUniqChar(String s) {
        for (int i = 0; i < s.length(); i++) {
            for (int j = 0; j < s.length(); j++) {
                if (i != j) {
                    if (s.charAt(i) == s.charAt(j)) {
                        break;
                    }
                }
                if (j == s.length() - 1) {
                    return i;
                }
            }
        }
        return -1;
    }
}